infra/scripts/release/bump_file_versions.py

```diff
@@ -33,46 +33,22 @@ def main() -> None:
     with open(path_to_file_list, "r") as f:
         files_to_bump = f.read().splitlines()
 
-    # The current version should be 0.18.0 or 0.19.0 or 0.20.0 etc, but we should also make sure to support the
-    # occasional patch release on the master branch like 0.18.1 or 0.18.2
-    versions_in_files = 0
-    if current_version[-2:] != ".0":
-        print(current_version[-2:])
-        versions_in_files = count_version(current_version, files_to_bump, repo_root)
-        if versions_in_files != VERSIONS_TO_BUMP:
-            raise SystemExit(f"Found {versions_in_files} occurrences of {current_version} in files to bump, but "
-                             f"expected {VERSIONS_TO_BUMP}")
-    else:
-        found = False
-
-        # Lets make sure the files don't contain a patch version (e.g, 0.x.0 -> 0.x.20)
-        for patch_version in range(0, 20):
-            current_version_patch = current_version[:-1] + str(patch_version)
-            versions_in_files = count_version(current_version_patch, files_to_bump, repo_root)
-
-            # We are using a patch version, let's change our version number
-            if versions_in_files == VERSIONS_TO_BUMP:
-                print(f"Found {versions_in_files} occurrences of {current_version_patch}, changing current version to "
-                      f"{current_version_patch}")
-                current_version = current_version_patch
-                found = True
-                break
-            else:
-                print(f"Found {versions_in_files} occurrences of {current_version_patch}, instead of {VERSIONS_TO_BUMP}")
-        if not found:
-            raise SystemExit(f"Could not find {VERSIONS_TO_BUMP} versions of {current_version} in {path_to_file_list}")
-
-    print(f"Found {versions_in_files} occurrences of {current_version} in files to bump {path_to_file_list}")
+    # The current version should be 0.18.0 or 0.19.0 or 0.20.0 etc
```

Comment:
Don't think we need 0.18.0 or 0.19.0 right? Also after version bump we're at 0.21?

Reply:
Just as an example, but I'll update.